config/config.go

```diff
@@ -68,11 +68,26 @@ type Build struct {
 	DiskSize    int
 	OEMSize     string
 	OEMFSSize4K uint64
+	SealOEM     bool
 	GPUType     string
 	Timeout     string
 	GCSFiles    []string
 }
 
+// Update clears the config file and then saves the new config.Build.
+func Update(configFile *os.File, buildConfig *Build) error {
```

Comment:
The name "Update" typically implies that the file isn't blindly overwritten, and is instead modified in a somewhat non-destructive way. Also, this function is specific to the build config (and does not work for other configs). For these reasons, I think it might be better to name this "SaveBuildConfigToFile" or "OverwriteBuildConfig".